apps/sms/style/sms.css

```diff
@@ -32,6 +32,23 @@ form[role="dialog"][data-type="confirm"] p {
   white-space: pre-line;
 }
 
+form[role="dialog"].error-dialog-show-recipient ul {
+  border: none;
+}
+
+form[role="dialog"].error-dialog-show-recipient ul li {
+  padding: 0;
+}
+
+.error-dialog-show-recipient li {
```

Comment:
there is already a comment a little above, I thought it was enough, but I can add another one here too.

I also prefered to keep the sole BB overrides in its own rule, and have what didn't need to be overriden in the simpler rule... This way we clearly see what we override and what we don't override. And I still hope that we can override in a simpler way in the future